src/interfaces/IMetaMorpho.sol

```diff
@@ -70,7 +70,8 @@ interface IMetaMorphoBase {
     function withdrawQueueLength() external view returns (uint256);
 
     /// @notice Stores the total assets managed by this vault when the fee was last accrued.
-    /// @dev May be a little off `totalAssets()` after each interaction, due to some roundings.
+    /// @dev May be greater than `totalAssets()` between deposits/withdrawals due to removal of markets with non-zero
+    /// supply or socialized bad debt. This difference will decrease the fee accrued until the next deposit/withdrawal.
```

Comment:
"between deposits/withdrawals" sounds like deposits&withdrawals cause the error. Also don't other functions (`redeem`, `mint`, `setFee`) fix `lastTotalAssets`?

```suggestion
    /// @dev May be greater than `totalAssets()` due to removal of markets with non-zero supply or socialized bad debt.
    /// This difference will decrease the fee accrued until the next deposit/withdrawal.
    /// This difference can be removed by calls to (...)
```